bin/node/cli/src/command.rs

```diff
@@ -14,33 +14,36 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-use sc_cli::{VersionInfo, error};
+use sc_cli::{VersionInfo, self};
```

Comment:
```suggestion
use sc_cli::VersionInfo;
```

Rust 2018 automatically makes crates usable everywhere without importing them explicitly.

Reply:
noted, thanks!